### ISSUE TITLE
Issue with version upgrade (v1.2.1)

### ISSUE THREAD

**Issue by soleny894** [Zibbp/ganymede#93]
Hi,

It seems you introduced arm64 builds in the last version but it's not containing anything for amd64 which create an issue for anyone using automated versioning upgrade. In my case, I accepted the update but my pod was kept back for this reason. Can you consider releasing a new version with both arm and amd64 support?
```
  Warning  Failed     3m10s (x4 over 4m40s)  kubelet            Failed to pull image "ghcr.io/zibbp/ganymede:v1.2.1": rpc error: code = NotFound desc = failed to pull and unpack image "ghcr.io/zibbp/ganymede:v1.2.1": no match for platform in manifest: not found
```

**Comment by Zibbp** [comment]
Hi,

For now the amd64 and arm64 builds are on separate tags. arm64 is on :latest and arm64 on :latest-arm64. Ideally I would like for them to be on the same tag and use the architecture option built into the repository but I was unable to get that functioning with the action workflow using two Dockerfiles. This is something I hope to fix soon.

I see you're using the v1.2.1 tag which is currently assigned to the arm64 image as it was the last one to be built so it was given the tag. For the time being you will have to use the :latest tag instead of targeting a version, if possible. I'll update this issue again when I have figured something out.

**Comment by soleny894** [comment]
Hi again,

I reverted back to v1.2.0 and blacklisted the v1.2.1 release so it's not proposed for update again.
I use renovate bot which check for new releases, it's better for me to use versioning than the latest tag because it's easier to keep going with releases and know which one is running on my cluster.
Merging both arch on the same tag would be great and I hope it will be a thing for next release.

**Comment by Zibbp** [comment]
The arm64 and amd64 images have been fixed in the latest v1.2.2 release. Now both the `latest` and `v*.*.*` tags have the multi-architecture images in the manifests.

Let me know if this works for you.

**Comment by soleny894** [comment]
Hi,

I can run the new v1.2.2 tag without any issue, thanks you!

Since it's fixed, I'll close this issue.
